src/feature/PDKFeature.ts

```diff
@@ -1,5 +1,5 @@
-'use strict';
```

Comment:
Do we need the strict?

Reply:
We should enable it in `tsconfig.json`, as right now its inconsistently added to the files across the project. But it probably shouldn't be removed in a feature add so I'll put it back